app/helpers/spree/admin/tax_categories_helper.rb

```diff
@@ -0,0 +1,8 @@
+module Spree::Admin::TaxCategoriesHelper
```

Comment:
We have a bunch of rubocop lint issues for this file. Can you please look into these? Thanks.